src/tools/clippy/clippy_lints/src/utils/inspector.rs

```diff
@@ -45,14 +46,10 @@ impl<'tcx> LateLintPass<'tcx> for DeepCodeInspector {
             return;
         }
         println!("impl item `{}`", item.ident.name);
-        match item.vis.node {
-            hir::VisibilityKind::Public => println!("public"),
-            hir::VisibilityKind::Crate(_) => println!("visible crate wide"),
```

Comment:
The special `pub(crate)` case should probably be kept here and below, it's important.